Makefile.am

```diff
@@ -59,10 +59,10 @@ hydra:	hydra.c $(OBJ)
 	@echo
 
 xhydra:	
-	-cd hydra-gtk && sh ./make_xhydra.sh
+	cd hydra-gtk && sh ./make_xhydra.sh
```

Comment:
if xhydra cannot be built then this should not result in a build failure. not everyone has gtk installed.